src/ready/apis/train_mobious.py

```diff
@@ -226,23 +228,20 @@ def main(args):
     transform_arg = transform_map.get(config.transforms.transform_operation, None)
     target_transform_arg = transform_map.get(config.transforms.target_transform_operation, None)
     
-
-    # Split datapath into trainpath, validation+path, test_path
-    # Create full mobious dataset first, then split into respective sets.
-    # Make sure to change dataset fed into trainloader
-    # use .random_split()
-    # Put validation into training loop
-    # Finally, evaluate with the test_set
-    
     ## Length 5; github_data_path
     ## Length 1143;  data_path
     full_dataset = MobiousDataset(
         data_path, transform=transform_arg ,target_transform=target_transform_arg
         )
     
     SEED = 42
+
+    TRAIN_SET_PROPORTION = config.datasets_proportions.train_set
+    VALIDATION_SET_PROPORTION = config.datasets_proportions.validation_set
+    TEST_SET_PROPORTION = config.datasets_proportions.test_set
+
     train_set, validation_set, test_set = torch.utils.data.random_split(full_dataset, 
-                                                                        [0.70, 0.15, 0.15],
+                                                                        [TRAIN_SET_PROPORTION, VALIDATION_SET_PROPORTION, TEST_SET_PROPORTION],
```

Comment:
DITTO. let's use `..._RATIO` for these variables as it is commonly used terminology.